blocks/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+# Create your models here.
+
+
+class Block(models.Model):
+    hash = models.CharField(max_length=100)
+    merkle_hash = models.CharField(max_length=100)
+    prev_blk_hash = models.CharField(max_length=100)
```

Comment:
- `blk` -> `block`
- can it be a relation to `Block`/`self` model?

Reply:
yup. done